kmos_tools/star_offsets.py

```diff
@@ -176,21 +181,20 @@ def star_fit_profile(exposure):
 
 
     """
-
+    #print('Y')
```

Comment:
please delete this line

Reply:
Whoops - sorry for missing so many of my print statements!